waterlog/src/Components/CostsReport.js

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react'
+import DailyCostsReports from './DailyCostsReports';
+import MonthlyCostsReports from './MonthlyCostsReports';
+import { fetchCostsDaily, fetchCostsMonthly, fetchCostsSeasonally } from '../actions/CostsReportActions';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
```

Comment:
' '

Reply:
I replaced some double quotes. Feature is agent and will be up for Demo. Will remove all in the next PR.

Reply:
...